includes/options/class-amp-options-manager.php

```diff
@@ -327,6 +328,45 @@ public static function update_analytics_options( $data ) {
 		return self::update_option( 'analytics', wp_unslash( $data ) );
 	}
 
+	/**
+	 * Renders the welcome notice on the 'AMP Settings' page.
+	 *
+	 * Uses the user meta values for the dismissed WP pointers.
+	 * So once the user dismisses this notice, it will never appear again.
+	 *
+	 * @todo: Add the full copy for this notice when it is decided.
+	 */
+	public static function render_welcome_notice() {
+		if ( 'toplevel_page_' . self::OPTION_NAME !== get_current_screen()->id ) {
+			return;
+		}
+
+		$notice_id = 'welcome-notice-1';
+		$dismissed = get_user_meta( get_current_user_id(), 'dismissed_wp_pointers', true );
+		if ( in_array( $notice_id, explode( ',', strval( $dismissed ) ), true ) ) {
+			return;
+		}
+
+		?>
+		<div class="notice notice-info is-dismissible" id="<?php echo esc_attr( $notice_id ); ?>"><p><?php esc_html_e( 'Welcome to AMP for WordPress', 'amp' ); ?></p></div>
+
+		<script>
+		( function( $ ) {
+			$( function() {
+				// On dismissing the notice, make a POST request to store this notice with the dismissed WP pointers so it doesn't display again.
+				$( document ).on( 'click', '#<?php echo $notice_id; // WPCS: XSS OK. ?> .notice-dismiss', function() {
```

Comment:
Instead of: `'#<?php echo $notice_id; // WPCS: XSS OK. ?> .notice-dismiss'` 
Do this: `<?php echo wp_json_encode( "#$notice_id .notice-dismiss" ); ?>`

Reply:
Thanks, https://github.com/Automattic/amp-wp/pull/1442/commits/8f02acf695c24ad6b5a2045a988e439564119381#diff-6dc720c5e5352d31e656ef4973984528R357 applies your suggestion.